doc/storage.md

```diff
@@ -13,9 +13,52 @@ are block devices that:
 * do not contain partition information
 * have more than 1 GiB
 
-Any such device will be prepared according to the value of `operator.satelliteSet.automaticStorageType`. Devices are
-added to a storage pool based on the device name (i.e. all `/dev/nvme1` devices will be part of the pool
-`nvme1`)
+To enable automatic configuration of devices, set the `devicePaths` key on `storagePools` entries:
```

Comment:
It may now make more sense to place this section under the "Configuring storage pool creation" section because this builds on that concept.